src/main/java/net/digibi/mailerlite/MailerLiteClient.java

```diff
@@ -12,34 +15,92 @@
     private static final String API_END_POINT = "https://api.mailerlite.com/api/v2/";
     private final String API_KEY;
 
+    /**
+     *
+     * @param apiKey Mailerlite apiKey required for mailerlite API
+     */
     public MailerLiteClient(String apiKey) {
         API_KEY = apiKey;
     }
 
-    public boolean subscribeUser(long groupId, String userBody) {
-        HttpResponse<String> response = Unirest
-            .post(API_END_POINT + "groups/" + groupId + "/subscribers")
-            .header("content-type", "application/json")
-            .header("x-mailerlite-apikey", API_KEY)
-            .body(userBody)
-            .asString();
+    /**
+     * Moves user provided in userBody to specified groupId
+     * @param groupId
+     * @param userBody
+     * @return
+     */
+    public boolean moveSubscriberToGroup(long groupId, String userBody) {
```

Comment:
`moveSubscriberToGroup` means remove a subscriber from an existing group to another group, and it's clearly not what this method is doing. If adding an existing subscriber to another group will have the side effect of removing it from its previous group, write it down in the Javadoc.